### ISSUE TITLE
import_playbook is invalid yaml as documented, does not work 

### ISSUE THREAD

**Issue by marvingreenberg** [ansible/ansible#28223]
##### ISSUE TYPE
 - ~B ug Report~
- Docs Report

##### COMPONENT NAME
import_playbook

##### ANSIBLE VERSION
```
ansible 2.3.2.0
  config file = /home/foo/.ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.13 (default, May  8 2017, 12:47:14) [GCC 4.4.7 20120313 (Red Hat 4.4.7-17)]
```
##### CONFIGURATION
Bog standard.  Set inventory path.
##### OS / ENVIRONMENT
RHEL6

##### SUMMARY
The documentation at (http://docs.ansible.com/ansible/latest/playbooks_reuse_includes.html) for **import_playbook** shows what seems to be invalid YAML.
```
---
import_playbook: webservers.yml
import_playbook: databases.yml
```
Sure enough, trying a simple example like this fails, with:
```
[WARNING]: While constructing a mapping from /data0/tenadarts/ansible/playbooks/configure.yml, line 2, column 1, found a duplicate dict key (import_playbook). Using last defined value only.
```
Followed by,
```
ERROR! playbooks must be a list of plays
```

##### STEPS TO REPRODUCE

Create two playbooks, test1.yml and test2.yml
```
---
- hosts: linux
  tasks:
    - ping: {}    
```
Create a playbook, pb.yml
```
---
import_playbook: test1.yml
import_playbook: test2.yml
```

Run:
```
ansible-playbook pb.yml
```

This results in the **ERROR! Playbooks must be a list of plays** above.
Trying to guess at the required structure, I also tried:
```
---
- import_playbook: test1.yml
- import_playbook: test2.yml
```
This yields: **ERROR! 'import_playbook' is not a valid attribute for a Play**

##### EXPECTED RESULTS
The two playbooks are imported then executed.

##### ACTUAL RESULTS

Various syntax errors.

**Comment by helins** [comment]
I have the exact same issue, I've tried fiddling with the syntax like you did and always got an error, even with version 2.0. It isn't specific to 2.3.

**Comment by resmo** [comment]
IMHO There is a missing "available in version 2.4" for this feature, see the changelog of 2.4 https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#24-dancing-days---active-development

**Comment by bcoca** [comment]
site is supposed to reflect 'latest stable' .. for some reason the 2.4 feature snuck in .. also it should be clearly labeled as a 2.4 feature and show the 'previous' way to do it.

**Comment by marvingreenberg** [comment]
Actually, there is still an issue.  Even as documented, it is invalid yaml.  If ansible is somehow working around the invalidity  -- non unique hash keys, see 1.3 about unique keys, (http://www.yaml.org/spec/1.2/spec.html) -- it shouldn't (IMO).

Instead it should use some approach that is still valid yaml, like:
```
    - import_playbook: playbook1.yml
    - import_playbook: playbook2.yml
    - hosts: all
        <...>
    - hosts: linux
        <...>
```
Essentially, import_playbook becomes a "play" whose only attribute is import_playbook.

**Comment by marvingreenberg** [comment]
OK, so I looked at the devel branch, and the implementation is correct, but the documentation is wrong.  The implementation is as described above, 
```
- import_playbook: foo.yml
- import_playbook: foo.yml
```
not as documented like this
```
import_playbook: foo.yml
import_playbook: foo.yml
```

There is one minor code related issue: in the event that a statement other than a play or an include/import_playbook is present, the exception thrown from (https://github.com/ansible/ansible/blob/devel/lib/ansible/playbook/__init__.py) at Line 90 says that the entries must be a 'play' or an 'include', but should say 'import_playbook' since include is deprecated.

**Comment by dagwieers** [comment]
Propose the changes in a PR. You can edit the required files through the Github interface, so no prior knowledge about Git required.

**Comment by marvingreenberg** [comment]
@bcoca Do you want to cherry pick the code changes related to the documentation changes?  Or just abandon them?

**Comment by adampbrush** [comment]
Make sure you are running 2.4, I had issues with 2.3.

https://controlaltfail.wordpress.com/2017/10/18/ansible-error-import-playbook-is-not-a-valid-attribute-for-a-play-master-playbook/

**Comment by racl101** [comment]
I encountered the:

`ERROR! playbooks must be a list of plays`

error myself and after double checking everything couldn't find the error. 

By accident I found that when I removed any trailing whitespace and/or newlines from my `playbook.yml` that **it fixed the issue**. 

It's weird because I tried validating my YAML with a linter before encountering this fix so I can't understand why it worked.

Admittedly, I don't have much experience with YAML so there might be some rule that I don't understand that I'm missing.
